src/LoginScreen.js

```diff
@@ -1,9 +1,61 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {loginWithUserData} from './helperFunctions/login';
+import {updateJWT} from './actions';
+import './stylesheets/login.css';
 
-let LoginScreen = () => 
-    <div>
-        <p>Login Screen</p>
-    </div>
 
 
+let mapStateToProps = (state) => {
+    return {jwt: state.jwt}
+};  
+
+let mapDispatchToProps = (dispatch) => {
+  return {dispatch: dispatch}
+};
+
+class LoginScreenDumb extends Component {
+
+    
+    render() {
+
+        let submitUserLoginInformation = (event) => {
+            event.preventDefault();
+            let username = event.target.username.value;
+            let password = event.target.password.value;
+            event.target.reset();
+            let JSONLoginInfo = {"username":username, "password":password}
+            loginWithUserData(JSON.stringify(JSONLoginInfo))
+            .then(res => this.props.dispatch(updateJWT(res)))
```

Comment:
Instead of this.props, once you have this hooked up to the store this is unnecessary and the deconstructed dispatch will be passed into the dumb component argument.  Ultimately, you can just do res => dispatch(updateJWT(token)).  FYI, I LOVE this pattern to use a function to create the dispatch object!  Totally stealing it.

Reply:
You are correct -- passing props into a functional component does not use the "this.props" statement; i used it since I created a class-based component.